### PR TITLE
Add option to show uncovered line contents

### PR DESCRIPTION
**Pull request by kasium**

### CODE REVIEW

**Comment by kasium** [comment]
Resolves #134

File: diff_cover/report_generator.py [hunk 1]
```diff
@@ -363,6 +364,11 @@ class StringReportGenerator(TemplateReportGenerator):
     """
 
     TEMPLATE_NAME = "console_coverage_report.txt"
+    INCLUDE_SNIPPETS = False
```
Comment (Bachmann1234) [inline diff_cover/report_generator.py:367]: IMO this format implies this is a constant and this very much is not.

I think this should just be a property on the object itself.

```
def __init__(self, violations_reporter, diff_reporter, show_uncovered=False):
        super().__init__(violations_reporter, diff_reporter)
        self.include_snippets = show_uncovered
```

**Comment by Bachmann1234** [comment]
Looking reasonable. Left one comment I feel fairly strongly about but I could be convinced otherwise. I know this is still a work in progress though

File: diff_cover/report_generator.py (hunk 1 above)
Comment (kasium) [reply diff_cover/report_generator.py]: The attribute is now in lowercase as well as the other twos (to keep it consitent)

**Comment by kasium** [comment]
@Bachmann1234 thanks for the comment. I renamed the constants. Tests are now also present

**Comment by Bachmann1234** [comment]
Awesome! Lets send this one out into the world

**Pull request merged**